PR: https://github.com/odoo/odoo/pull/210844

From: 21037d91e3f2afa5d426abe38d9187a5f7d4d4c9
From: clesgow
Date: 2025-05-23 23:04:15

Breaking data model changes scores: del:3 + add:0, change matches:
-    posx = fields.Integer('Corridor (X)', default=0, help="Optional localization details, for information purpose only")
-    posy = fields.Integer('Shelves (Y)', default=0, help="Optional localization details, for information purpose only")
-    posz = fields.Integer('Height (Z)', default=0, help="Optional localization details, for information purpose only")

Total Changes: 3

[REM] stock: remove location positions

These fields are completely unused, and even considering those as
"purely informative", they were removed from the location form since
[1].

[1] 9410e0c

closes odoo/odoo#210844

Related: odoo/upgrade#7711
Signed-off-by: William Henrotin (whe) <whe@odoo.com>

================================= pseudo patch: =================================

--- a/addons/stock/models/stock_location.py
+++ b/addons/stock/models/stock_location.py
@@ -60,9 +60,6 @@ class StockLocation(models.Model):
         help='This location (if it\'s internal) and all its descendants filtered by type=Internal.'
     )
     comment = fields.Html('Additional Information')
-    posx = fields.Integer('Corridor (X)', default=0, help="Optional localization details, for information purpose only")
-    posy = fields.Integer('Shelves (Y)', default=0, help="Optional localization details, for information purpose only")
-    posz = fields.Integer('Height (Z)', default=0, help="Optional localization details, for information purpose only")
     parent_path = fields.Char(index=True)
     company_id = fields.Many2one(
         'res.company', 'Company',
